inst/rmarkdown/templates/xaringan/resources/ki-fonts.css

```diff
@@ -0,0 +1,12 @@
+body {
+  font-family: system, -apple-system, ".SFNSText-Regular", "San Francisco", "Roboto", "Segoe UI", "Helvetica Neue", "Lucida Grande", sans-serif;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: Cambria, "Hoefler Text", Utopia, "Liberation Serif", "Nimbus Roman No9 L Regular", Times, "Times New Roman", serif;
+  font-weight: 400;
+}
+
+.remark-code, .remark-inline-code {
+  font-family: Consolas, "Andale Mono WT", "Andale Mono", "Lucida Console", "Lucida Sans Typewriter", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Liberation Mono", "Nimbus Mono L", Monaco, "Courier New", Courier, monospace;
+}
```

Comment:
- Do you really need that many fonts?

- Seems like you are missing the `@import` call to import the fonts?

Reply:
I think the font is fine. it doesn't import the fancy online fonts, the code will use whatever is found in the listed fonts.

Reply:
A valid concern is that the font appearance may not be consistent across different OS/browsers. I would suggest the same as @pat-s mentioned above, using the @import fonts.

An example from @emitanaka : https://github.com/emitanaka/ninja-theme/blob/master/docs/themes/kunoichi/assets/default-fonts.css

Reply:
> I think the font is fine. it doesn't import the fancy online fonts, the code will use whatever is found in the listed fonts.

True, this might not be a practical problem but I see possible confusion for users questioning themselves if they need all these fonts to use the theme properly.

I also did that in the past but nowadays only use the fonts I also import 😄 . This makes it more clear what is happening.